setup.py

```diff
@@ -1,25 +1,35 @@
-from setuptools import setup
+from setuptools import setup, find_packages
 
 #####################################
 VERSION = "0.0.0"
-ISRELEASED = False 
+ISRELEASED = False
 if ISRELEASED:
     __version__ = VERSION
 else:
     __version__ = VERSION + '.dev0'
 #####################################
 
 
+requirements = [
+    'numpy',
+    'scipy',
+    'mbuild>=0.10.4',
+    'unyt>=2.4',
+    'boltons',
+    'lxml',
+]
+
```

Comment:
General question - does listing this array replace the use of a `requirements.txt` file?

Reply:
No, and I recommend the use of `install_requires` if you had to pick one. The reasons for using each are slightly different. The actual dependencies (things that should be installed when the package is installed) should go in `setup.py` in the `install_requires`. See these links for more info:
1. https://www.reddit.com/r/Python/comments/3uzl2a/setuppy_requirementstxt_or_a_combination/
2. https://packaging.python.org/discussions/install-requires-vs-requirements/